### PR TITLE
Remove address-form dependency and fix build

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?

Fixes the `npm run build` command, which wasn't working because rollup couldn't figure out the exports of the `lodash` package.

Also removes the unnecessary `address-form` dependency, and inline the `removeValidation` function to maintain compatibility.

#### What problem is this solving?

The build wasn't working before, and the published NPM package is outdated.

I tried to publish a new version with the changes from `master`, but since the build wasn't working I couldn't do it. After this PR, I think we should be good to go.

#### How should this be manually tested?

Run unit tests.

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/utils.js [hunk 1]
```diff
@@ -181,3 +181,20 @@ export function findSlaWithChannel(item, channel) {
...
+      value.value == null
+        ? typeof value === 'object'
+          ? null
+          : value
+        : value.value
+
+    return [key, newValue]
+  })
+
+  const newAddress = Object.fromEntries(newAddressEntries)
```
Comment (lucasecdb) [inline react/utils.js:197]: these methods are [polyfilled on checkout](https://github.com/vtex/vcs.checkout-ui/blob/master/src/index.html#L21), so we should be good 👀

Comment (lucasecdb) [reply react/utils.js:197]: actually, only `Object.entries` is polyfilled... should we add `Object.fromEntries` as well?

**Review by jeffersontuc — APPROVED** [review]

File: react/utils.js (hunk 1 above)
Comment (jeffersontuc) [reply react/utils.js:197]: hmm nice catch! I think we should 🤔

Comment (lucasecdb) [reply react/utils.js:197]: created here https://github.com/vtex/vcs.checkout-ui/pull/1322

**Pull request merged**